custom/icds_reports/templates/icds_reports/icds_app/awc-reports.html

```diff
@@ -71,7 +71,7 @@ <h2 class="center">{$ $ctrl.data.chart[0].key $}</h2>
                 </div>
             </div>
             <div class="row black" ng-if="$ctrl.step === 'beneficiary'">
-                <div ng-if="$ctrl.showTable" class="col-md-12">
+                <div ng-if="$ctrl.showTable && $ctrl.isAWCsSelected()" class="col-md-12">
```

Comment:
since the same two conditions are applied in three places could you make it a helper function on the controller?

that would also hopefully increase the likelihood of us not messing this up again in the future.

Reply:
added